writefile.go

```diff
@@ -8,6 +8,11 @@ import (
 	"golang.org/x/xerrors"
 )
 
+// WriteTupleFileEncodersToFile generates array backed MarshalCBOR and UnmarshalCBOR implementations for the
+// given types in the specified file, with the specified package name.
+//
+// The MarshalCBOR and UnmarshalCBOR implementations will marshal/unmarshal each type's fields as a
+// fixed-length array of field values.
```

Comment:
Maybe "tuple" instead of "array"?

Reply:
Well, CBOR calls them arrays. I'll change it to "CBOR array".